resources/js/components/data-list/Actions.js

```diff
@@ -56,7 +56,7 @@ export default {
                     });
                 }
 
-                this.$emit('completed');
+                this.$emit('completed', response.data);
```

Comment:
The first argument is the success boolean (see the one happening in the failure) so it should be `true`, then the response data.

Reply:
Ah ok, I guess that was implied as it's not explicitly passed here. Adding it explicitly...